experimental/Schemes/src/ToricBlowups/attributes.jl

```diff
@@ -1,80 +1,77 @@
 export exceptional_prime_divisor
 
 @doc raw"""
-    underlying_morphism(bl::ToricBlowupMorphism)
+    underlying_morphism(phi::ToricBlowupMorphism) -> ToricMorphism
 
 Return the underlying toric morphism of a toric blowup. Access to other
 attributes such as `domain`, `codomain`, `covering_morphism` are
 executed via `underlying_morphism`.
 
 # Examples
 ```jldoctest
-julia> P3 = projective_space(NormalToricVariety, 3)
+julia> X = projective_space(NormalToricVariety, 3)
 Normal toric variety
 
-julia> f = blow_up(P3, [0, 1, 1])
+julia> phi = blow_up(X, [0, 1, 1])
 Toric blowup morphism
 
-julia> Oscar.underlying_morphism(f)
+julia> Oscar.underlying_morphism(phi)
```

Comment:
Maybe just?

```suggestion
julia> underlying_morphism(phi)
```

The corresponding function is exported, and hopefully works. Added benefit, since we likely do not want to advertise internal functions to users.

Reply:
I don't think `underlying_morphism` is exported. Should I export it?

Reply:
Ok. I stand corrected - it is indeed not exported. I see two options:
1. Export it and document it.
2. Do keep it not-exported, but then also do not document it.

@HechtiDerLachs Unless you feel strongly about it, I suggest to export it.

Reply:
No, it was never the intention to have this function exported. It is an internal function used to facilitate our inheritance patterns. If you want to get rid of the toric structure, we decided to implement some method of `forget_toric_structure` for that. Maybe that can be used in the doctest, too.

Reply:
In this case, it feels that we should just remove the docstring. No point in having a docstring for this function, if we do not export it. (`forget_toric_structure` should have a separate docstring IMHO.)

Reply:
Commenting out the docstring for a private function (@HereAround's commit https://github.com/oscar-system/Oscar.jl/pull/4566/commits/5aa754cb9e59864ea4330e29195bcceab088257d) is a harmless change here. But I would like to add my thoughts about docstrings of private functions in general. In Python, if a functions does not have a docstring, there should be at least a comment indicating what the function does in the place where a docstring would usually be in [1]. In the default settings, the most common Python linter requires all functions to have a docstring [2]. The only exceptions are init functions (in Julia, these would be the creator functions in structs).

[1] https://peps.python.org/pep-0008/#documentation-strings
[2] https://pylint.pycqa.org/en/latest/user_guide/messages/convention/missing-function-docstring.html

In general it would be a lot better if JuliaFormatter would handle all formatting questions so there would be no time wasted style discussions in pull requests. OSCAR guidelines say nothing about docstrings for private functions. If we as a community do not want docstring for private functions, then this should be discussed in a separate issue/meeting and then added to JuliaFormatter so that this is applied everywhere uniformly.